src/openrct2/interface/console.c

```diff
@@ -390,9 +390,11 @@ void console_scroll(sint32 linesToScroll)
     }
 }
 
+// Calculates the amount of visible lines, based on the console size, excluding the input line.
 static sint32 console_get_num_visible_lines()
 {
-    return ((_consoleBottom - 22 - _consoleTop) / 10) - 1;
+    sint32 lineHeight = 10;
```

Comment:
`const`

Reply:
Ok. As this will be the last correction (unless you see something else to fix), would you care if I cleaned up the commit history along with this? Just to merge the corrections into the appropriate commits